File: lab02 /src/main/java/com/example/lab2/domain/Post.java
package com.example.lab2.domain;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.Setter;

import javax.persistence.*;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Entity
public class Post {

   @Id
   @GeneratedValue(strategy = GenerationType.IDENTITY)
   long id;
   private String title;
   private String content;
   private String author;




}

File: lab7/src/components/Post/Post.js

import React from "react";
import './Post.css'
const Post = (props) => {
  
    return (
          
        <div className="Content">
              <div className="Field">
               <div><p>Id:{props.id}</p></div>
                <div><p>Title:{props.title}  </p></div>
                <div><p>Author:{props.author} </p></div>
            </div>
         </div>
         
    );

    

    
}

export default Post;